### PR TITLE
Add cost to table, display inv. ref. and addr., change pdf name

### PR DESCRIPTION
**Pull request by aanil**
Up on stage

### CODE REVIEW

File: run_dir/static/js/pricing_main.js [hunk 1]
```diff
@@ -379,10 +379,10 @@ const vPricingMain = {
             axios
                 .get(url)
                 .then(response => {
-                    this.USD_in_SEK = response.data.USD_in_SEK
-                    this.EUR_in_SEK = response.data.EUR_in_SEK
+                    this.USD_in_SEK = response.data.USD_in_SEK.toFixed(2)
+                    this.EUR_in_SEK = response.data.EUR_in_SEK.toFixed(2)
```
Comment (alneberg) [inline run_dir/static/js/pricing_main.js:383]: I don't think we want this right? Keeping the decimals is a good thing.

File: run_dir/static/js/pricing_main.js [hunk 2]
```diff
@@ -379,10 +379,10 @@ const vPricingMain = {
...
                     date = new Date(Date.parse(response.data['Issued at']))
-                    this.exch_rate_issued_at = date.toISOString().substring(0, 10)
+                    this.exch_rate_issued_at = date.toLocaleDateString('se-SE')
```
Comment (alneberg) [inline run_dir/static/js/pricing_main.js:385]: Very good catch!

File: run_dir/static/js/pricing_main.js [hunk 3]
```diff
@@ -583,15 +583,15 @@ app.component('v-exchange-rates', {
           if (val === null) {
               return ""
           } else {
-              return val.toFixed(2)
+              return val
```
Comment (alneberg) [inline run_dir/static/js/pricing_main.js:586]: I think these two should be added back as they were as well.

File: run_dir/static/js/pricing_main.js (hunk 1 above)
Comment (aanil) [reply run_dir/static/js/pricing_main.js]: Done!

File: run_dir/static/js/pricing_main.js (hunk 3 above)
Comment (aanil) [reply run_dir/static/js/pricing_main.js]: Done!

**Review by alneberg — APPROVED** [review]
Perfect, thank you!

**Pull request merged**